src/components/Dashboard/TasksColumns/TaskColumns.scss

```diff
@@ -0,0 +1,31 @@
+.tasksColumnToDoTitle {
+  margin: 25px auto 1px;
+  text-align: center;
+  font-size: 1.8rem;
+  color: rgb(103, 102, 106);
+  border-bottom: 3px solid rgb(103, 102, 106);
+}
+
+.tasksColumnInProgressTitle {
+  margin: 25px auto 1px;
+  text-align: center;
+  font-size: 1.8rem;
+  color: rgb(118, 183, 192);
+  border-bottom: 3px solid rgb(116, 154, 159);
+}
+
+.taskColumnDoneTitle {
+  margin: 25px auto 1px;
+  text-align: center;
+  font-size: 1.8rem;
```

Comment:
please add additional class  `.taskColumn` for repeatable rules or group existing classes
`.taskColumnDoneTitle, .tasksColumnInProgressTitle, .tasksColumnToDoTitle { common rules here}`

Reply:
Thank you for the review, I’ll correct everything as needed :)